src/reactive-graphql.ts

```diff
@@ -67,12 +68,24 @@ function isFieldWithResolver(
 
 export default function graphql<T = object>(
   schema: Schema,
```

Comment:
```
graphql(
  schema: GraphQLSchema,
  requestString: string,
  rootValue?: ?any,
  contextValue?: ?any,
  variableValues?: ?{[key: string]: any},
  operationName?: ?string
)
```

Actually, 3rd parameter is supposed to be `rootValue`. If you support to `rootValue` in the future while being similar to the reference signature, the API should be break. I think you should add unused `rootValue` parameter now.

Reply:
Yeah, this makes sense, I'll try to add it in this PR 👍